client/state/selectors/get-p2-hub-blog-id.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import isSiteWPForTeams from 'calypso/state/selectors/is-site-wpforteams';
+
+/**
+ * Returns the P2 hub blog id for the site. If the site is a hub the id will be
+ * the same as the site id.
+ *
+ * @param {object} state Global state tree
+ * @param {number} siteId the site ID
+ * @returns {?number} Blog ID of the site's hub if it is a P2 – otherwise null.
+ */
+export default function getP2HubBlogId( state, siteId ) {
+	if ( ! isSiteWPForTeams( state, siteId ) ) {
+		return null;
+	}
+	return get( state, [ 'sites', 'items', siteId, 'options', 'p2_hub_blog_id' ], 0 );
```

Comment:
Could you update the `0` to `null` as well since the default bad case is `null`?

Reply:
Good catch! 👍